test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -592,28 +594,8 @@ void test_ucp_mmap::release_exported_memh_buf(void *exported_memh_buf)
     ucp_memh_buffer_release(exported_memh_buf, &release_params);
 }
 
-void test_ucp_mmap::import_memh(void *exported_memh_buf, ucp_mem_h *memh_p)
-{
-    ucp_mem_map_params_t params;
-
-    params.field_mask           =
-            UCP_MEM_MAP_PARAM_FIELD_EXPORTED_MEMH_BUFFER;
-    params.exported_memh_buffer = exported_memh_buf;
-
-    {
-        scoped_log_handler warn_slh(import_no_md_error_handler);
-        ucs_status_t status = ucp_mem_map(receiver().ucph(), &params, memh_p);
-        if (status == UCS_ERR_UNREACHABLE) {
-            release_exported_memh_buf(exported_memh_buf);
-            UCS_TEST_SKIP_R("memory importing is unsupported");
-        }
-        ASSERT_UCS_OK(status);
-    }
-}
-
-void test_ucp_mmap::test_rereg_imported_mem(ucp_mem_h memh,
-                                            uint64_t memh_pack_flags,
-                                            size_t size)
+ucp_mem_h test_ucp_mmap::import_memh(ucp_mem_h exported_memh,
+                                     uint64_t memh_pack_flags)
```

Comment:
seems this method is always packign exported key, so `memh_pack_flags` is not really nedeed

Reply:
ok, true, I'll remove this param from this function and 2 others